sympy/__main__.py

```diff
@@ -0,0 +1,357 @@
+#!/usr/bin/env python
```

Comment:
This line should not be here. The `__main__.py` file shouldn't be executed directly (it generally won't work).

Reply:
You could also add a comment to the top of this file indicating that this is the `isympy` script.